cal/cal/static/js/app.js

```diff
@@ -52,7 +52,12 @@ function TagListCtrl($scope, $http, CalendarFilterService, TagService) {
   };
 
   this.create = function(tag) {
-    TagService.createTag(tag.label, tag.keywords)
+    var filterData = CalendarFilterService.getFilter();
+    var start = this.isCumulative ? null : filterData.start;
+    var end = this.isCumulative ? null : filterData.end;
```

Comment:
I think you should just use `filterData.start` since you're not doing anything with the intermediate variable